services/core-web/src/components/Home.js

```diff
@@ -109,4 +124,4 @@ export class Home extends Component {
 
 Home.propTypes = propTypes;
 
-export default AuthenticationGuard(Home);
+export default connect(null)(AuthenticationGuard(Home));
```

Comment:
is the connect needed here?

Reply:
It is. Connect gives me access to the dispatch function, which then lets me dispatch each action without having to bind each action creator by name.